packages/plugin-e2e/src/models/GrafanaPage.ts

```diff
@@ -9,6 +9,16 @@ import { PluginTestCtx } from '../types';
 export abstract class GrafanaPage {
   constructor(public readonly ctx: PluginTestCtx, protected readonly expect: Expect<any>) {}
 
+  protected async navigate(url: string, options?: NavigateOptions) {
```

Comment:
Planning to use this method across all models, but will do that in a separate PR.